Packages/vcs/Lib/configurator.py

```diff
@@ -510,12 +447,13 @@ def reset_template_changes(state):
           logo_button.set_state(1)
 
     def setup_animation(self):
-        if self.initialized == False:
+        if self.initialized is False and [True for d in self.displays if d.g_type not in ("marker", "text", "textcombined", "fillarea", "line", "textorientation", "texttable")]:
```

Comment:
We don't want to enable animation controls for displays that aren't animate-able, as this causes VCS to throw a (perhaps well-deserved) hissy-fit. Sorry for the gnarly list comprehension; I can move that out to a function if we care particularly.